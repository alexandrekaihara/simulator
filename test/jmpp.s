LOAD ONE
JMPP ISGRATERZERO
OUTPUT MINUSONE
STOP
ISGRATERZERO:
LOAD MINUSONE
JMPP ISLESSZERO
OUTPUT ONE
STOP
ISLESSZERO:
OUTPUT MINUSONE
STOP
ONE: CONST 1
MINUSONE: CONST -1
