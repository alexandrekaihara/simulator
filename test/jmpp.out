10 17 7 7 13 18 14 10 18 7 14 13 17 14 13 18 14 1 -1